src/routers/handlers/removeCompanyHandler.ts

```diff
@@ -0,0 +1,98 @@
+import { Request, Response } from "express";
```

Comment:
This file is at the wrong level. It should be inside yourCompanies folder, so the path should be src/routers/handlers/yourCompanies/removeCompanyHandler.ts

Also it has very low test coverage. The red numbers indicate lines of code not covered by unit tests at all
![Screenshot 2024-10-03 at 11 24 23](https://github.com/user-attachments/assets/a5ef1719-c172-4514-903a-c500b0a17361)

Reply:
We improved this coverage by refactoring the code to get rid of unnecessary errors, what do you think about it now?

Reply:
SonarQube indicates that the parts of the code that deal with errors are not covered with tests at all
![Screenshot 2024-10-03 at 16 27 02](https://github.com/user-attachments/assets/d51cc9f4-ebff-4bfa-b8ad-4b4e02efea2e)